backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/location/impl/AddressBuilder.java

```diff
@@ -487,7 +487,10 @@ public String buildBPartnerFullAddressString(
 			return "";
 		}
 
-		final String displaySequence = extractDisplaySequence(bpLocation, location);
+		final boolean isLocal = isLocalCountry(bpLocation);
+		final String displaySequence = getDisplaySequence(location.getC_Country(), isLocal);
```

Comment:
pls use CountryDAO to get the country.
avoid model getters like `location.getC_Country()`

Reply:
(we already have countriesRepo as a field)